<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filetool</class>
 <widget class="QMainWindow" name="filetool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>filetool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Go to</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item>
             <widget class="QTreeView" name="treeView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QListView" name="listView">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
               <property name="modelColumn">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QTableView" name="tableView">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="showGrid">
                <bool>true</bool>
               </property>
               <property name="gridStyle">
                <enum>Qt::NoPen</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QListView" name="iconView">
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
               <property name="viewMode">
                <enum>QListView::IconMode</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_new">
     <property name="title">
      <string>&amp;new</string>
     </property>
     <addaction name="actionFile"/>
     <addaction name="actionDirectory"/>
    </widget>
    <addaction name="menu_new"/>
    <addaction name="action_open"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="actionRe_do"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="actionC_opy"/>
    <addaction name="action_paste"/>
    <addaction name="separator"/>
    <addaction name="action_delete"/>
    <addaction name="action_rename"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionList_Details_View"/>
    <addaction name="actionList_View"/>
    <addaction name="actionIcon_View"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_exit"/>
   <addaction name="separator"/>
   <addaction name="actionGo_home"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="actionParentDir"/>
   <addaction name="separator"/>
   <addaction name="action_undo"/>
   <addaction name="actionRe_do"/>
   <addaction name="separator"/>
   <addaction name="action_delete"/>
   <addaction name="action_cut"/>
   <addaction name="actionC_opy"/>
   <addaction name="action_paste"/>
   <addaction name="separator"/>
   <addaction name="action_open"/>
   <addaction name="action_rename"/>
   <addaction name="separator"/>
   <addaction name="actionFile"/>
   <addaction name="actionDirectory"/>
   <addaction name="separator"/>
   <addaction name="actionList_View"/>
   <addaction name="separator"/>
   <addaction name="actionList_Details_View"/>
   <addaction name="actionIcon_View"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionC_opy">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>c&amp;opy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/edit-undo.png</normaloff>:/images/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRe_do">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/edit-redo.png</normaloff>:/images/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>re&amp;do</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_rename">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/edit-rename.png</normaloff>:/images/edit-rename.png</iconset>
   </property>
   <property name="text">
    <string>&amp;rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>file</string>
   </property>
   <property name="toolTip">
    <string>create file</string>
   </property>
  </action>
  <action name="actionDirectory">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/folder-new.png</normaloff>:/images/folder-new.png</iconset>
   </property>
   <property name="text">
    <string>directory</string>
   </property>
   <property name="toolTip">
    <string>create dir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;open</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/dialog-information.png</normaloff>:/images/dialog-information.png</iconset>
   </property>
   <property name="text">
    <string>about...</string>
   </property>
  </action>
  <action name="actionList_View">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/view-list-details.png</normaloff>:/images/view-list-details.png</iconset>
   </property>
   <property name="text">
    <string>List View</string>
   </property>
  </action>
  <action name="actionIcon_View">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/view-list-icons.png</normaloff>:/images/view-list-icons.png</iconset>
   </property>
   <property name="text">
    <string>Icon View</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>previous</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </action>
  <action name="actionParentDir">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/go-up.png</normaloff>:/images/go-up.png</iconset>
   </property>
   <property name="text">
    <string>parentDir</string>
   </property>
  </action>
  <action name="actionGo_home">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/go-home.png</normaloff>:/images/go-home.png</iconset>
   </property>
   <property name="text">
    <string>go_home</string>
   </property>
   <property name="toolTip">
    <string>home</string>
   </property>
  </action>
  <action name="actionList_Details_View">
   <property name="icon">
    <iconset resource="filetool.qrc">
     <normaloff>:/images/view-list-text.png</normaloff>:/images/view-list-text.png</iconset>
   </property>
   <property name="text">
    <string>List Details View</string>
   </property>
   <property name="toolTip">
    <string>List Details</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="filetool.qrc"/>
 </resources>
 <connections/>
</ui>
